src/cli.rs

```diff
@@ -94,26 +95,90 @@ pub struct IcebreakersConfig {
 
 impl Config {
     pub fn from_args(args: Args) -> Result<Self, AppError> {
-        let network_magic = Self::get_network_magic(&args.network);
-        let icebreakers_config = match (args.solitary, args.reward_address, args.secret) {
-            (false, Some(reward_address), Some(secret)) => Some(IcebreakersConfig {
-                reward_address,
-                secret,
-            }),
+        let server_address = match env::var("SERVER_ADDRESS") {
+            Ok(val) => val,
+            Err(_) => args.server_address,
```

Comment:
So after this is merged, env vars will have priority over CLI arguments – is this fine? /cc @gytis-ivaskevicius